src/civis/response.py

```diff
@@ -230,9 +223,7 @@ def __hash__(self):
 
     def _to_dataclass_repr_pprint(self):
         """Convert the response to a dataclass for repr and pprint purposes."""
-        # Only show the "Response" class name at the outermost level.
-        class_name = "" if self._inner_response else "Response"
-        klass = dataclasses.make_dataclass(class_name, self._data_snake.keys())
+        klass = dataclasses.make_dataclass("Response", self._data_snake.keys())
```

Comment:
I was trying to make the repr look nicer with less indentation by not showing the class name "Response" in embedded objects, but in my actual usage, not showing it for embedded objects had actually make it harder to read :-(